build.gradle.kts

```diff
@@ -610,9 +610,17 @@ tasks {
     destinationDirectory.set(projectDir.dir("distribution"))
   }
 
+  val assembleSparkConnector by register("assembleSparkConnector", Copy::class) {
+    dependsOn("spark-connector:shadowJar")
+    group = "gravitino distribution"
+    finalizedBy("checksumSparkConnector")
+    from("spark-connector/build/libs/${rootProject.name}-spark-connector-runtime-$version.jar")
```

Comment:
Will this package include the kyuubi jar?

Reply:
I prefer to add it then user no need to have to download it, ok, i will add it.

Reply:
done, add scala version to pacakge name like `gravitino-spark-connector-runtime-3.4_2.12-0.5.0-SNAPSHOT.jar`